index.html

```diff
@@ -69,125 +82,134 @@ <h1 id="headline-primary">
-                </p>
-                <ul class="art-printing-list">
-                    <li>html</li>
-                    <li>bootstrap</li>
-                    <li>Ruby</li>
-                </ul>
+            <div class="art-printing bg-5">
+                <div class="art-printing-inner">
+                    <h3 class="art-printing-heading">Profesional Art Printing Data</h3>
+                    <p class="art-printing-description">
+                        A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+                    </p>
+                    <ul class="art-printing-list">
+                        <li>html</li>
+                        <li>bootstrap</li>
+                        <li>Ruby</li>
+                    </ul>
+                </div>
+                <a class="btn-lg" href="#">See Project</a>
+            </div>
+
+            <div class="art-printing bg-6">
+                <div class="art-printing-inner">
+                    <h3 class="art-printing-heading">Profesional Art Printing Data</h3>
+                    <p class="art-printing-description">
+                        A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+                    </p>
+                    <ul class="art-printing-list">
+                        <li>html</li>
+                        <li>bootstrap</li>
+                        <li>Ruby</li>
+                    </ul>
+                </div>
+                <a class="btn-lg btn-dark-yellow" href="#">See Project</a>
             </div>
-            <a class="btn-lg btn-dark-yellow" href="#">See Project</a>
         </div>
     </section>
 
     <section class="about-section padding-left-right">
         <div class="about-box">
-            <h2 class="about-heading">About me</h2>
-            <p class="about-description">
-                Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
-            </p>
-            <a class="btn-sm" href="#">Get my Resume</a>
+            <div class="about-text-box">
+                <h2 class="about-heading">About me</h2>
+                <p class="about-description">
+                    Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+                </p>
+                <a class="btn-sm" href="#">Get my Resume</a>
+            </div>
 
             <div class="about-img"></div>
         </div>
 
-        <hr />
+        <hr class="about" />
 
         <div class="about-categories">
             <div class="categories">
```

Comment:
The 2nd category should say `Frameworks`, not `Language` in order to match the technologies listed below. Please fix the header.